src/app/pages/questions/components/question/slider-input/slider-input.component.scss

```diff
@@ -22,5 +22,23 @@ slider-input {
 
   ion-label {
     @include font-large;
+
+    padding-bottom: 0;
+  }
+
+  .padding {
+    margin: -10px 5px;
+    font-size: 80%;
+  }
+
+  .labelLeft {
+    float: left;
+    width: 50%;
+  }
+
+  .labelRight {
+    float: right;
+    width: 50%;
```

Comment:
If both are 50% does that mean they will be very close together when the length of the text is equal to max width?
If so, might be better to make them a bit less like 48% each.

Reply:
Good point, made the change.